src/main/java/edu/harvard/iq/dataverse/api/Admin.java

```diff
@@ -1920,4 +1926,56 @@ public Response getBannerMessages(@PathParam("id") Long id) throws WrappedRespon
 
     }
     
+    @GET
+    @Path("/licenses")
+    public Response getLicenses() {
+        return ok(licenseService.listAll().stream()
+        		.map(JsonPrinter::json)
+        		.collect(toJsonArray()));
+    }
+
+    @GET
+    @Path("/licenses/{id}")
+    public Response getLicense(@PathParam("id") long id) {
+    	License l = licenseService.get(id);
+    	if (l == null) {
+    		return error(Response.Status.NOT_FOUND, "Not Found.");
+		}
```

Comment:
Indent properly.

Reply:
GitHub displays tabs incorrectly, so I'm changing them to spaces.

Reply:
As discussed, we will use 4 spaces per indent. That seems to be roughly the rule.